README.md

```diff
@@ -137,6 +137,12 @@ To check available options:
 lucca angular update --help
 ```
 
+To update a specific version of Lucca Front : 
+
+```
+npx ng update @lucca-front/ng@16.5.0
```

Comment:
We could add that `npx ng update @lucca-front/ng` is the same as `npx ng update @lucca-front/ng@latest` and you can also run `npx ng update @lucca-front/ng@rc` (the @ points either to a specific version or a npm release channel ("tags"))

Reply:
to understand, if you run `npx ng update @lucca-front/ng@rc` it will update the latest rc ?

Reply:
yes it will target this version 

![image](https://github.com/LuccaSA/lucca-front/assets/2089620/e401d324-5c91-4f38-b6fa-28c7e48cdaa9)

see https://www.npmjs.com/package/@lucca-front/ng?activeTab=versions

Reply:
thanks @shajz i update the doc !